dist/DatePicker/DatePickerInput.js

```diff
@@ -31,23 +31,32 @@ function _objectWithoutProperties(source, excluded) { if (source == null) return
 
 function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }
 
+var defaultInputProps = {
+  autoComplete: 'off'
```

Comment:
this camelized case threw me off
see: https://reactjs.org/docs/dom-elements.html